terraform/install.sh

```diff
@@ -37,6 +38,7 @@ promptForBillingAccount() {
     log ""
     log "To list active billing accounts, run:"
     log "gcloud beta billing accounts list --filter open=true"
+    python telemetry.py event=no-active-billing version=$VERSION
```

Comment:
All the executions of your telemetry script from within the same install.sh (or terraform) need to be correlated. I.e. we need to know that creation of the sandbox, and then loadgen and all other events are part of the same larger event of a user trying to create a new Sandbox. You should generate a UUID and then use it in all the executions of the telemetry script within the same session of running install.sh

Reply:
That's a good point. I will do that.